frontend/src/components/modals/accountDelete/DeleteAccountModal.tsx

```diff
@@ -28,6 +29,13 @@ const DeleteAccountModal = ({ hideModal }: IModal) => {
                     Anuluj
                 </PrimaryButton>
             </DeleteAccountButtonsWrapper>
+            {error && (
+                <ErrorIndicator>
+                    {error.response
```

Comment:
Response can be 'undefined' and throw error here, i send you good way to do it with '.hasOwnProperty()' on DC.

Check all other place where u have <ErrorIndicator>